build/debian-iptables/Makefile

```diff
@@ -19,12 +19,13 @@ IMAGE=debian-iptables
 TAG=v5
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
+QEMUVERSION=v2.7.0
```

Comment:
if we're bumping the qemu version, then we probably should bump the tag, too. (also given that arm is changing base images.)